core/src/test/kotlin/org/evomaster/core/database/SqlInsertBuilderTest.kt

```diff
@@ -184,7 +184,7 @@ class SqlInsertBuilderTest {
 
         val genes = actions[0].seeGenes()
         assertEquals(1, genes.size)
-        assertTrue(genes[0] is SqlTimestampGene)
+        assertTrue(genes[0] is DateTimeGene)
```

Comment:
shouldn't also add a check on the format of the date gene being the one expected for SQL?

Reply:
agree. Added more assertions to check all the expected behaviour of such dateTime genes